src/database/build.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN;
+DROP TABLE IF EXISTS users, consultant,posts CASCADE;
+
+
+CREATE TABLE users(
+    id SERIAL  PRIMARY KEY  ,
+    user_name VARCHAR(60) NOT NULL,
+    password TEXT NOT NULL
+
```

Comment:
@alaabadra Hi! Do you think users should have a `full_name` field?  I'm not sure if it's an important feature, but I noticed that the `consultant` table had that field so it might be good to keep both tables consistent.